CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+## 0.49.0
+September 6, 2018
+
+## ⚠️ Breaking changes
+* Use `client{Height/Width}` instead of `offset{Height/Width}` for map canvas sizing ([#6848](https://github.com/mapbox/mapbox-gl-js/issues/6848), fixed by [#7128](https://github.com/mapbox/mapbox-gl-js/pull/7128))
+
+## 🐛 Bug fixes
+* Fix inadvertent purging of feature state by requiring manual feature state reset ([#7141](https://github.com/mapbox/mapbox-gl-js/pull/7141))
+* Fix Top Issues list for mapbox-gl-js ([#7108](https://github.com/mapbox/mapbox-gl-js/issues/7108), fixed by [#7112](https://github.com/mapbox/mapbox-gl-js/pull/7112))
+* Fix bug in which symbols with `icon-allow-overlap: true, text-allow-overlap: true, text-optional: false` would show icons when they shouldn't ([#7041](https://github.com/mapbox/mapbox-gl-js/pull/7041))
+* Fix bug in `Map#flyTo` where the map wouldn't stop on an exact zoom level when the final zoom value is not the one requested by `flyTo` (#7222) ([#7223](https://github.com/mapbox/mapbox-gl-js/pull/7223)) (h/t [benoitbzl](https://github.com/benoitbzl))
```

Comment:
I think this should just be `Fix bug where the map would not stop at the exact zoom level requested by Map#FlyTo`